es6-classes/circle.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+
+class Circle extends Shape {
+  constructor(radius) {
+    super();
+    this.radius = radius;
+    this.area = this.radius * this.radius * Math.PI;
+    this.perimeter = this.radius * Math.PI * 2;
+  }
+
+  describe() {
+    return `The area of the shape is ${this.area}.
+    The perimeter of the shape is ${this.perimeter}.
+    The radius of the shape is ${this.radius}.`;
+  }
```

Comment:
Instead of logging `this.area` & `this.perimeter`, you should be calling the `describe` method of `super`